requirements.txt

```diff
@@ -0,0 +1,3 @@
+datasets==2.16.1
```

Comment:
Add line  `--extra-index-url https://download.pytorch.org/whl/cpu`  if `torch` is needed